reference/form_fields.sf.yml

```diff
@@ -123,10 +123,29 @@ components:
           description: The minimum valid value for the field (integer and date type only).
         maxLength:
           type: integer
-          description: The maximum length for the value (string type only)
+          description: The maximum length for the value (string type only).
         secret:
           type: boolean
-          description: Whether the field represents a password field
+          description: Whether the field represents a password field (password field type only).
```

Comment:
Do these changes only apply to the `custom` password fields or also the `built-in` password fields?

Reply:
I think it's both, since there's a [field](https://github.com/bigcommerce/docs/blob/8bd4e672333e19fe8f83b07a6bd9447d30ab52ba/reference/form_fields.sf.yml#L90)in the response body that talks about both built-in and custom. I could be wrong though.

Reply:
I think its for both as well, so this change probably still makes sense

Reply:
Sounds good, @bc-rowan @bc-akshatkaushik could someone approve my PR?